packages/mask/src/plugins/CryptoartAI/apis/index.ts

```diff
@@ -0,0 +1,118 @@
+import { OrderSide } from 'opensea-js/lib/types'
+import type { ChainId } from '@masknet/web3-shared-evm'
+import { resolveAPILinkOnCryptoartAI } from '../pipes'
+
+export async function getAsset(tokenId: string, chainId?: ChainId) {
+    const ownersResponse: any = await (
+        await fetch(resolveAPILinkOnCryptoartAI(chainId) + '/api/artwork/current/owners/' + tokenId, {
```

Comment:
plz use https://npm.im/urlcat build all request urls

maybe, you can make a general url build function.

e.g:
```
makeCryptoartAPI('/artwork/current/owners/:tokenId', { tokenId })
```

Reply:
Fixed.

Reply:
Added `urlcat `